spec/stoplight/admin_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Stoplight::Admin, type: %i[request] do
+  let(:light) { Stoplight("foo") }
+  let(:light_condition) { proc { 1 / 1 == 0 } }
+
+  before do
+    redis_instance = Redis.new(url: ENV.fetch("STOPLIGHT_REDIS_URL", "redis://127.0.0.1:6379/0"))
```

Comment:
I think you can add `:redis`, metadata to provide redis instance and database cleaner:

```ruby
RSpec.describe Stoplight::Admin, type: %i[request], redis: true do
  Stoplight.configure do |config|
      config.data_store = Stoplight::DataStore::Redis.new(redis)
  end
end
```

Reply:
Yep, it's much cleaner this way 🙂